apis/capabilities/v1beta1/product_types.go

```diff
@@ -702,6 +732,10 @@ type ProductSpec struct {
 	// ProviderAccountRef references account provider credentials
 	// +optional
 	ProviderAccountRef *corev1.LocalObjectReference `json:"providerAccountRef,omitempty"`
+
+	// Policies holds the product's policy chain
+	// +optional
+	Policies []PolicyConfig `json:"policies,omitempty"`
```

Comment:
Is PolicyConfig not being a pointer intentional?

Reply:
I see no reason to use pointers.